app/models/user.rb

```diff
@@ -7,4 +7,8 @@ class User < ApplicationRecord
   validates_presence_of :first_name
   enum role: %i[default admin]
   has_secure_password
+
+  def bookmarkings(user)
+    user.videos.includes(:tutorial).order("tutorials.title")
```

Comment:
I think since this is in the User model we might not have to call user in this method, I think the class is the implicit receiver if nothing else is called. We can see what self is in this method and see if I'm right or not.